core/src/main/java/org/apache/carbondata/core/scan/result/vector/impl/directread/AbstractCarbonColumnarVector.java

```diff
@@ -130,4 +131,9 @@ public CarbonColumnVector getDictionaryVector() {
   public void convert() {
     // Do nothing
   }
+
+  @Override
+  public void setLazyPage(LazyPageLoad lazyPage) {
+    throw new UnsupportedOperationException("Not allowed from here");
```

Comment:
Put the class name in the message, it is easier for debugging

Reply:
ok